dist/Select/Select.d.ts

```diff
@@ -1,38 +1,65 @@
-export default Select;
-declare function Select({ disabled, hint, invalid, label, onChange, options, placeholder, required, value, }: {
-    disabled: any;
-    hint: any;
-    invalid: any;
-    label: any;
-    onChange: any;
-    options: any;
-    placeholder: any;
-    required: any;
-    value: any;
+import React from 'react';
+declare type OptionType = {
+    label: string;
+    value: number | string;
+};
+export default function Select(props: {
+    /**
+     * If true, the element is not interactive.
```

Comment:
oooh does having comments in the type definitions mean that prop descriptions will show up in TS editor hints?

Reply:
it does!

Reply:
<img width="779" alt="Screen Shot 2020-12-16 at 12 48 35 PM" src="https://user-images.githubusercontent.com/739304/102386423-09194b00-3f9d-11eb-91f7-b726206d160f.png">

Reply:
Cool!